spec/helpers/link_to_helper_spec.rb

```diff
@@ -146,4 +146,32 @@
 
   end
 
+  describe "when constructing a google analytics event string" do
+    it "returns correctly formatted event javascript" do
+      expect(analytics_event_code("test category", "test")).to eq(
+        "if (pageTracker) { pageTracker._trackEvent('test category','test') };"
+      )
+    end
+
+    context "when passed a label value" do
+      it "includes the default value 1 if none is supplied" do
+        expect(analytics_event_code("test category", "test", "test label")).to eq(
+          "if (pageTracker) { pageTracker._trackEvent('test category','test','test label',1) };"
+        )
+      end
+
+      it "uses the supplied value if there is one" do
+        expect(analytics_event_code("test category", "test", "test label", 42)).to eq(
+          "if (pageTracker) { pageTracker._trackEvent('test category','test','test label',42) };"
+        )
+      end
+
+      it "doesn't wrap the label in quotes if passed label_is_script=true" do
+        expect(analytics_event_code("test category", "test", "window.location.href", 42, true)).to eq(
+          "if (pageTracker) { pageTracker._trackEvent('test category','test',window.location.href,42) };"
```

Comment:
Why might I want to use `window.location.href` as the 3rd param vs `'test label'` as in the previous spec above? Presumably the former is going to dynamically grab the window URL; Is it valid to supply a String?